test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -228,8 +228,8 @@ Date: REGEX(.*)
     "id": "REGEX([0-9a-f]{24})",
     "notification": {
         "attrs": [
-            "humidity",
-            "temperature"
+            "temperature",
```

Comment:
> About pretty-print ... I thought the order in vectors weren't touched by the pretty-print ...

That's right... but I don't refere to the pretty print of the GET subscriptions operation, but the pretty print of the accumulator, in which the attributes came in a key-map where the pretty printer will act. For this particular.test (the one in which this commen has been inserted) that comment doesn't really applies (as no accumulator server is started) but maybe for others .test in which attrs order has been also changed it applies (I haven't reviewed in detail...).